packages/nest-commander-testing/src/command-test.factory.ts

```diff
@@ -7,6 +7,7 @@ import {
   CommandRunnerService,
   Inquirer,
 } from 'nest-commander';
+import { CommanderOptionsType } from 'packages/nest-commander/src/command-factory.interface';
```

Comment:
Ah, missed this, CI caught it, you should change this line to just import from `nest-commander`

```suggestion
import { CommanderOptionsType } from 'nest-commander';
```

If it doesn't exist in `nest-commander'`s `index.ts` root export, then you'll need to add it